protocol/messenger_communities.go

```diff
@@ -5145,3 +5145,46 @@ func (m *Messenger) PromoteSelfToControlNode(communityID types.HexBytes) (*commu
 
 	return community, nil
 }
+
+func (m *Messenger) createResponseWithACNotification(communityID string, acType ActivityCenterType) (*MessengerResponse, error) {
+	// Activity center notification
+	notification := &ActivityCenterNotification{
+		ID:          types.FromHex(uuid.New().String()),
+		Type:        acType,
+		Timestamp:   m.getTimesource().GetCurrentTime(),
+		CommunityID: communityID,
+		Read:        false,
+		Deleted:     false,
+		UpdatedAt:   m.GetCurrentTimeInMillis(),
+	}
+
+	response := &MessengerResponse{}
+
+	err := m.addActivityCenterNotification(response, notification, nil)
+	if err != nil {
+		m.logger.Error("failed to save notification", zap.Error(err))
+		return response, err
+	}
+
+	return response, nil
+}
+
+func (m *Messenger) RegisterOwnerTokenReceivedNotification(communityID string) (*MessengerResponse, error) {
```

Comment:
Just suggestion: you can reduce this PR to one function, if you pass `acType ActivityCenterType` in rpc arguments from nim